main/coursecopy/recycle_course.php

```diff
@@ -82,18 +114,40 @@
     if (!$course->has_resources()) {
         echo get_lang('NoResourcesToRecycle');
     } else {
-        echo Display::return_message(get_lang('RecycleWarning'), 'warning', false);
+         echo Display::return_message(get_lang('RecycleWarning'), 'warning', false);
         $form = new FormValidator('recycle_course', 'post', api_get_self().'?'.api_get_cidreq());
         $form->addElement('header', get_lang('SelectOptionForBackup'));
         $form->addElement('radio', 'recycle_option', null, get_lang('FullRecycle'), 'full_backup');
         $form->addElement('radio', 'recycle_option', null, get_lang('LetMeSelectItems'), 'select_items');
+
+        //Confirmation input code
+        $form->addElement('label', '', '<span class="hidden course-full-delete">'.get_lang('CourseCodeConfirmation').'</span>', null, null);
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 140 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5721344)